quarkus/test-commons/src/main/java/org/apache/polaris/test/commons/PostgresRelationalJdbcLifeCycleManagement.java

```diff
@@ -53,19 +56,40 @@ public Map<String, String> start() {
 
     context.containerNetworkId().ifPresent(postgres::withNetworkMode);
     postgres.start();
-    return Map.of(
-        "polaris.persistence.type",
-        "relational-jdbc",
-        "quarkus.datasource.db-kind",
-        "pgsql",
-        "quarkus.datasource.jdbc.url",
-        postgres.getJdbcUrl(),
-        "quarkus.datasource.username",
-        postgres.getUsername(),
-        "quarkus.datasource.password",
-        postgres.getPassword(),
-        "quarkus.datasource.jdbc.initial-size",
-        "10");
+
+    if (separateDs) {
```

Comment:
is this really useful for this class, which manages the database container? Would it not be clearer for it to have the list of databases to pre-configure?